django/db/migrations/operations/base.py

```diff
@@ -111,6 +111,15 @@ def allow_migrate_model(self, connection_alias, model):
 
         return router.allow_migrate_model(connection_alias, model)
 
+    def reduce(self, operation, in_between, app_label=None):
+        """
+        Either returns a list of zero, one or two operations, meaning the
```

Comment:
Most of the sentence was borrowed from the removed `MigrationOptimizer.reduce` method. I'll try to come up with something the makes more sense.